### PR TITLE
Make explorer geo more optimized

### PR DESCRIPTION
**Pull request by ahuang11**
1. Batch call watchers so the loading symbol doesn't flicker (just tabbed everything over)
2. If scatter/line, swap to points/path
3. Projection doesn't always seem to be provided, so use crs instead

### CODE REVIEW

File: hvplot/ui.py [hunk 1]
```diff
@@ -701,6 +708,7 @@ def _plot(self):
             self._alert.param.update(
                 object=f'**Rendering failed with following error**: {e}', visible=True
             )
+            raise e
```
[the remark this answers is not in the document]
Comment (ahuang11) [reply hvplot/ui.py]: Intentional while debugging :) but not meant to be pushed.

File: hvplot/ui.py [hunk 2]
```diff
@@ -669,10 +675,11 @@ def _plot(self):
                 xmax = np.max(np.abs(self.xlim()))
                 self.geographic.crs = 'PlateCarree' if xmax <= 360 else 'GOOGLE_MERCATOR'
                 kwargs['crs'] = self.geographic.crs
+            if 'projection' not in kwargs:
+                kwargs['projection'] = kwargs['crs']
```
[the remark this answers is not in the document]
Comment (ahuang11) [reply hvplot/ui.py]: I guess not with this guard
https://github.com/holoviz/hvplot/pull/1353/commits/c4c1c7522ac0a56d913ab13df443575a84d45ec5#diff-e8e2b619a805d78b88c365301a48695ea26e0f325c0662ff25af3e791b90be86R679

**Review by maximlt — APPROVED** [review]

**Review by maximlt — CHANGES_REQUESTED** [review]
Ah in fact there are two failing tests:

```
FAILED hvplot/tests/testui.py::test_explorer_geo_revise_kind[kind_tuple0] - TypeError: Constant parameter 'geo' cannot be modified
FAILED hvplot/tests/testui.py::test_explorer_geo_revise_kind[kind_tuple1] - TypeError: Constant parameter 'geo' cannot be modified
```

**Comment by ahuang11** [comment]
New test failure; prob should handle in a separate PR.
`AttributeError: The geopandas.dataset has been deprecated and was removed in GeoPandas 1.0. You can get the original 'naturalearth_cities' data from [https://www.naturalearthdata.com/downloads/110m-cultural-vectors/.](https://www.naturalearthdata.com/downloads/110m-cultural-vectors/)`

**Review by maximlt — APPROVED** [review]

**Pull request merged**